docs/dev_tools/build_profiles.md

```diff
@@ -5,11 +5,15 @@ mbed OS 5 supports three primary build profiles: *develop*, *debug* and *release
 * Small and fast code.
 * Full error information. For example, asserts have file name and line number.
 * Hard to follow code flow when using a debugger.
+* Chip is put to sleep when going idle:
```

Comment:
Query: Could we change this to "Chip goes to sleep we idle" and keep the meaning? I would like to avoid the passive voice in "is put to sleep".